wiki/Article_Style_Guide/en.md

```diff
@@ -131,10 +157,26 @@ You can find a list of native language names at [Language names in their own lan
 
 ### Section Headings
 
-All section headings are to use title case, just like the article titles.
+All section headings are to use title case, just like the [article titles](#article-titles).
 
 All section headings are to be using levels 2 to 5 headings.
 
+Section headings should **never** skip a heading level.
+For example: do not go from a level 2 heading to a level 4 heading.
+
+All sections **must** contain some content, even those that act like a container for subsections.
+
+```
+## Game Modes
+
+_Also see [Glossary](/wiki/Glossary)._
+
+### osu!
+
+osu! is a frustrating but fun circle-clicking simulator.
+
+```
+
 Section headings must **never** have a link in it.
 Instead place the link underneath the section heading.
 For example:
```

Comment:
but that wouldn't ​make any sense, this is an example of how you should do it, not an example of what not to do